plugins/premium/advtable.md

```diff
@@ -44,8 +44,12 @@ tinymce.init({
 
 {% include misc/plugin-menu-item-id-boilerplate.md %}
 
+## Events
```

Comment:
My bad, should have spotted this earlier, but can you move the events section to after the commands section?